src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -6,9 +6,15 @@
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import java.math.BigDecimal;
-import lombok.Data;
-
-@Data
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
```

Comment:
the problem with data is that it includes this annotation, so you didn't resolve the problem
if you don't need equals at this point, just remove it